packages/rke2-canal/charts/templates/rbac.yaml

```diff
@@ -16,6 +15,14 @@ rules:
       - namespaces
     verbs:
       - get
+  # EndpointSlices are used for Service-based network policy rule
+  # enforcement.
+  - apiGroups: ["discovery.k8s.io"]
+    resources:
+      - endpointslices
+    verbs:
+      - watch
+      - list
```

Comment:
I'm always unsure - are those changes synced from upstream or something you do yourself?

If it's upstream - why we have to copy the whole yaml files instead of consuming the tarball?

Reply:
Copied from upstream but adding helm variable to make values configurable through values.yaml. In rbac.yaml, everything is copied as it is. The problem is that canal upstream does not have a helm chart that we can consume directly

Reply:
The way to install canal (https://docs.projectcalico.org/getting-started/kubernetes/flannel/flannel#installing-calico-for-policy-and-flannel-aka-canal-for-networking) is by executing `curl https://docs.projectcalico.org/manifests/canal.yaml -O`, that one is a manifest, i.e. it hardocdes the values of everything

Reply:
That's unfortunate, but I guess we have to live with that :/